### PR TITLE
Concert page

### PR DESCRIPTION
**Pull request by Shakir-Hussain12**
**For this PR, I**
---
- Set Up Linters and ESLint checks.
- Created & Styled the Index Page.
- Created & Styled the About Page.
- Used Javascript DOM to manipulate the navbar, features speakers, and programs section.
- Added 'Show More' & 'Show Less' buttons to featured speakers.
- Used Flexbox to maintain elements.
- Made both pages responsive for all screens.
- Deployed the project to GitHub Pages.
- Ran accessibility checks on Lighthouse for better content
- Loom Link : https://www.loom.com/share/8706018a2539463e994174aaf2f63618

### CODE REVIEW

File: assets/stylesheets/style.css [hunk 1]
```diff
@@ -0,0 +1,652 @@
...
+}
+
+footer div {
+  max-width: 90%;
+  margin: 0 auto;
+  display: flex;
+  justify-content: space-evenly;
+  align-items: center;
+  padding: 2%;
+}
```
Comment (Gopxfs) [inline assets/stylesheets/style.css:281]: To follow the original design, please make sure that your footer, in the mobile version of your homepage, has a white background:
image

File: assets/stylesheets/style.css [hunk 2]
```diff
@@ -0,0 +1,652 @@
...
+  padding: 0;
+}
+
+.top {
+  display: none;
+}
+
+nav {
+  background: linear-gradient(to right, lightyellow, white);
+}
```
Comment (Gopxfs) [inline assets/stylesheets/style.css]: As requested in the project requirements:
image

Please make sure to follow the original design colors. Keep in mind that your webpage doesn't need to be identical, but remember that you are following design guidelines! This project's goal is to improve your ability to follow design guidelines (imagine Microverse as a client requesting a website) 😉. 

For this reason, kindly remove the gradients from the headers and update it background colors to look accordingly to the original idea:

Your design:
image

- Top header: black gradient background and black text
- Bottom header: yellow gradient background

Original design:
image

- [x] Top header: solid black background and white text
- [x] Bottom header: solid white background

**Review by Gopxfs — CHANGES_REQUESTED** [review]
# ♻️ STATUS: REQUIRED CHANGES ♻️

## Hi @Shakir-Hussain12 👋,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights :rocket:
✔️ No linter errors.
✔️ Great code structure. Well done!
✔️ Nice theme!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 🗒️  
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Gopxfs in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Gopxfs** [comment]
https://github.com/Shakir-Hussain12/Capstone1/blob/concert-page/README.md?plain=1#L112-L114

As requested on the project requirements: 
image 

You should add a link to your video presenting the project on this PR 😉. Currently, you have a Loom video presenting your Portfolio project, instead of the capstone.

**Comment by Gopxfs** [comment]
https://github.com/Shakir-Hussain12/Capstone1/blob/concert-page/README.md?plain=1#L239-L245

As requested in the project requirements:  
image  

Please make sure to give credit to the [original author](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) of the design 😉.  
> Note: the credits should be in the `Acknowledgments` section

**Comment by Gopxfs** [comment]
https://github.com/Shakir-Hussain12/Capstone1/blob/concert-page/README.md?plain=1#L265-L270

Make sure to add the corresponding path to your MIT since it isn't working: :wink:: 
image

> Please remove all the empty sections from your `README` file!

**Review by Kossi-stack — APPROVED** [review]
## Status: Approved ✔️

**Hi @Shakir-Hussain12,**

You did a really great job! 💯 💯💯💯
Congratulations! 🎉 🎉 🎉 🎉 

128382030-38e7de91-f6f2-45cf-b54e-8068e0446e4c

You may close this Pull Request and merge the changes
Keep being awesome dev's and good luck with your next milestone!

## To Highlight

✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏 👏 👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**